app/themes/default/views/layouts/default.html.slim

```diff
@@ -43,9 +43,12 @@ html
                 ul.dropdown-menu
                   li
                     = link_to(t('user.admin.navbar.account_settings'), edit_user_registration_path)
-                  - if can?(:manage, current_tenant)
+                  - if can?(:manage, :all)
                     li
                       = link_to(t('layout.navbar.admin_panel'), admin_path)
+                  - if can?(:manage, current_tenant)
```

Comment:
I guess it might be often if I'm also a instance admin ?  To handle stuffs like role requests.

But won't be a big issue, I'll update code for now and see how users say